tests/launcher/test_local.py

```diff
@@ -217,8 +218,15 @@ def test_run_local_serving_job(batching, batch_size, code_to_function):
     inputs = {"data": str(input_csv_path)}
     params = {"batching": batching, "batch_size": batch_size}
 
-    result = project.run_function(job, inputs=inputs, params=params, local=True)
-    responses = result.status.results["return"]
+    from pathlib import Path
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        temp_path = str(Path(temp_dir))
```

Comment:
1. You may use pytest's built-in (`tmp_path`)[https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture] fixture
2. Please move the import statement to the top level